tasks/managed/add-fbc-contribution/add-fbc-contribution.yaml

```diff
@@ -66,14 +68,19 @@ spec:
         #
         set -eo pipefail
 
-        SNAPSHOT_PATH=$(workspaces.data.path)/$(params.snapshotPath)
-        RESULTS_FILE="$(workspaces.data.path)/$(params.resultsDirPath)/add-fbc-contribution-results.json"
+        # receives the updated version of the snapshotPath
+        SNAPSHOT_PATH="$(workspaces.data.path)/$(params.snapshotPath).updated"
         DATA_FILE="$(workspaces.data.path)/$(params.dataPath)"
         if [ ! -f "${DATA_FILE}" ] ; then
             echo "No valid data file was provided."
             exit 1
         fi
 
+        # adding a new result as modifying the current one used breaks e2e for single component.
+        # to be handled in RELEASE-1640.
+        RESULTS_FILE="$(workspaces.data.path)/$(params.resultsDirPath)/internal-requests-results.json"
+        echo -n "$RESULTS_FILE" > "$(results.internalRequestResultsFile.path)"
```

Comment:
This seems wrong. It should be the relative path inside the workspace dir, not full absolute path.

Reply:
changed.

Reply:
What do you think about doing this?
```
RESULTS_FILE="$(params.resultsDirPath)/internal-requests-results.json"
echo -n "$RESULTS_FILE" > "$(results.internalRequestResultsFile.path)"
RESULTS_FILE=$(workspaces.data.path)/${RESULTS_FILE}
```
That way you wouldn't have to add the prefix when using the variable later. But your current version is ok too. Just a suggestion.

Reply:
This is done.